web/templates/admin/list_key.html

```diff
@@ -0,0 +1,71 @@
+    <div class="l-center">
+      <div class="l-sort clearfix noselect">
+        <div class="l-unit-toolbar__buttonstrip">
+        <?php
+        	 echo '<a href="/add/key/" id="btn-create" class="ui-button cancel" title="'.__('Add SSH Key').'"><i class="fas fa-plus-circle status-icon green"></i>'.__('Add SSH Key').'</a>';
+        ?>
+      </div>
+     </div>
+    </div>
+    
+<div class="l-separator"></div>
+
+<div class="l-center units animated fadeIn">
+  <div class="header table-header">     
+    <div class="l-unit__col l-unit__col--right">
+      <div class="clearfix l-unit__stat-col--left wide-3"><b><?php print __('SSH_ID');?></b></div>
+      <div class="clearfix l-unit__stat-col--left compact-2">
+        &nbsp;
+      </div> 
+      <div class="clearfix l-unit__stat-col--left wide-7"><b><?php print __('SSH KEY');?></b></div>
+    </div>
+  </div>
+
+ <?php
+ 	$i = 0;
+    foreach ($data as $key => $value) {
+    ++$i;
+  ?>
+    <div class="l-unit header">
+      <div class="l-unit__col l-unit__col--right">
+        <div class="clearfix l-unit__stat-col--left wide-3"><b><?=$data[$key]['ID'];?></b></div>
+<div class="clearfix l-unit__stat-col--left text-left compact-2">
+                      <div class="l-unit-toolbar__col l-unit-toolbar__col--right noselect">
+                          <div class="actions-panel clearfix">
+                            <div class="actions-panel__col actions-panel__delete shortcut-delete" key-action="js">
+                              <a id="delete_link_<?=$i?>" class="data-controls do_delete" title="<?=__('delete')?>">
+                                <i class="fas fa-trash status-icon red status-icon dim do_delete"></i>
+                                <input type="hidden" name="delete_url" value="/delete/key/?key=<?=$key?>&token=<?=$_SESSION['token']?>" />
+                                <div id="delete_dialog_<?=$i?>" class="confirmation-text-delete hidden" title="<?=__('Confirmation')?>">
+                                  <p class="confirmation"><?=__('DELETE_KEY_CONFIRM',$key)?></p>
+                                </div>
+                              </a>
+                            </div>
+              
+                          </div>
+                        </div>
+                  </div>
+        <div class="clearfix l-unit__stat-col--left wide-7"><b><?=substr($data[$key]['KEY'], 0,50).'.....'.substr($data[$key]['ID'], -10,10);?></b></div>
```

Comment:
escape all variables that are rendered in the browser eg: htmlspecialchars($str, ENT_QUOTES)